src/pages/app-development/app-submission-guidelines.md

```diff
@@ -39,27 +37,52 @@ If your submission is rejected:
 
 The following requirements must be met for your app to be accepted. If your app does not meet these requirements, it will be rejected.
 
+To ensure that your submission's documentation contains the expected information, we recommend using relevant sections of the [Commerce Integration Starter Kit documentation](https://github.com/adobe/commerce-integration-starter-kit/blob/main/README.md) or the [Commerce Checkout Starter Kit documentation](https://github.com/adobe/commerce-checkout-starter-kit/blob/main/README.md) as a starting point.
+
+Some requirements will differ based on whether you indicated the app was non-downloadable during the submission process. This is because non-downloadable apps are installed in auto-generated environments, and users do not have access to the project files.
+
 ### Documentation
 
+- General documentation guidelines
+  - **Project-specific language**: Use project-specific language. Avoid generic references to "Adobe", "Magento" or "starter kit".
+  - **Configuration details**: Clearly list the app configuration and any third-party service-specific setup steps.
+  - **Clear summary**: Provide a compact description that captures what the app’s purpose and primary use cases.
+  - **Usage instructions**: Add instructions on how to use the app post-installation.
+  - For non-downloadable apps:
+      - All installation steps and usage instructions need to be included outside of the app submission's files at the application documentation URL provided in Exchange. Documentation within the app's files will not be accessible to users.
```

Comment:
```suggestion
      - All installation steps and usage instructions must be posted at the application documentation URL provided in Exchange. Because the app is not downloadable, README files and similar documents will not be accessible to users.
```